### DISCUSSION TITLE
miryoku-configurator-crkbd.json not compiling on QMK configurator

### DISCUSSION THREAD

**Discussion by jgeeliss** [manna-harbour/miryoku#251] [Q&A] [answered]
Hi,

When I try to compile file <data/configurator/miryoku-configurator-crkbd.json> on config.qmk.fm I get error "gmake: *** ap_introspection.o] Error 1".

Any idea why? Maybe an old keycode that is no longer supported?

**Comment by manna-harbour** [comment]
That file is very old so I wouldn't be surprised if it's no longer compatible. It's not supported at all.

`cpp` to produce a single `keymap.c` followed by `qmk c2json` might work for importing into the configurator. However, I performed a quick test and while the `cpp` and `c2json` outputs looks ok, the import into the configurator was silently unsuccessful. Perhaps it doesn't handle the `TD` and will need some preprocessing to remove them.

For Miryoku customisation see https://github.com/manna-harbour/miryoku/discussions/85.

**Comment by jgeeliss** [comment, marked as answer]
ah, found the issue, apparently the keymap name cannot have a hyphen in it. When I change 
"keymap": "manna-harbour_miryoku",
to:
"keymap": "manna_harbour_miryoku",
then it builds just fine!
